extensions/openvpn/src/utils.ts

```diff
@@ -1,32 +1,123 @@
-import { exec } from "child_process";
+import { exec, execSync } from "child_process";
+import { runAppleScript } from "run-applescript";
 import { promisify } from "util";
 
 const execAsync = promisify(exec);
 
 export const isRunning = async () => {
   try {
-    const { stdout, stderr } = await execAsync('ps u $(pgrep -vu root) | grep "OpenVPN" | grep -v "grep"');
-    return stdout.length > 0 || !!stderr;
+    const result = await runAppleScript(`
+      try
+        tell application "System Events"
+          tell process "OpenVPN Connect"
+            -- Check if the menu bar item 1 of menu bar 2 exists
+            if exists menu bar item 1 of menu bar 2 then
+              return true
+            else
+              return false
+            end if
+          end tell
+        end tell
+      on error
+        return false
+      end try
+    `);
+
+    return result === "true";
   } catch (e) {
     return false;
   }
 };
 
 export const startOpenVPN = async () => {
   try {
-    await execAsync('"/Applications/OpenVPN Connect/OpenVPN Connect.app/contents/MacOS/OpenVPN Connect"');
+    execSync('"/Applications/OpenVPN Connect/OpenVPN Connect.app/contents/MacOS/OpenVPN Connect"');
+    execSync('"/Applications/OpenVPN Connect/OpenVPN Connect.app/contents/MacOS/OpenVPN Connect" --minimize');
```

Comment:
why do we need it twice?

Reply:
Ah this could have been worth a little comment there..

In a [previous update](https://github.com/raycast/extensions/commit/3685d21bae36df2b411ce3f1fb0222c1b8d16644#diff-db0f5bd5fef7e5e97a25225fc79b4d00c0478382600c3ef8ff0edf415b639519R3) you remove the flag, because it would prevent the app from starting. In deed the minimize works when the app is already open. So we start and then minimize it.. Weird code but this way we do achieve the "start minimised" 🙂 

I'll add a comment though 👍